### PR TITLE
[Issue Template] Update for pod env & ask for project

### PR DESCRIPTION
**Pull request by segiddins**
\c @mrackwitz @neonichu @orta

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -22,12 +22,13 @@
 > ℹ Please replace this with of what happened instead.  
 > e.g. Pod A is missing the subspec B for target C.
 
-## Podfile
+## CocoaPods Environment
 
-> ℹ Please paste the contents of your Podfile below.    
->  (e.g. via `cat Podfile | pbcopy`)  
+> ℹ Please replace this with the output of `pod env`.
```
Comment (neonichu) [inline .github/ISSUE_TEMPLATE.md]: Might make sense to retain the hint on `pbcopy`

Comment (segiddins) [inline .github/ISSUE_TEMPLATE.md]: done

**Comment by segiddins** [comment]
This might also now fix the spurious integration spec failures

**Comment by segiddins** [comment]
It passed 😍

**Comment by orta** [comment]
Woooo0

**Pull request merged**